lib/ansible/modules/cloud/amazon/rds_param_group.py

```diff
@@ -84,17 +75,24 @@
   immediate:
     description:
       - Whether to apply the changes immediately, or after the next reboot of any associated instances.
-    required: false
-    default: null
-    aliases: []
+    aliases:
+      - apply_immediately
   params:
     description:
       - Map of parameter names and values. Numeric values may be represented as K for kilo (1024), M for mega (1024^2), G for giga (1024^3),
         or T for tera (1024^4), and these values will be expanded into the appropriate number before being set in the parameter group.
-    required: false
-    default: null
-    aliases: []
-author: "Scott Anderson (@tastychutney)"
+  tags:
+    description:
+      - Dictionary of tags to attach to the parameter group
+    version_added: "2.4"
+  purge_tags:
+    description:
+      - Whether to remove tags that aren't passed in the C(tags) parameter
+    default: no
+    version_added: "2.4"
+author:
```

Comment:
Is this different to the previous behaviour? If it's not, I'm not too keen to fix this now, I'd happily support someone else's PR to add `purge_params`.